content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -62,7 +62,7 @@ kubectl config view
 # get the password for the e2e user
 kubectl config view -o jsonpath='{.users[?(@.name == "e2e")].user.password}'
 
-kubectl config view -o jsonpath='{.users[].name}'    # get a list of users
+kubectl config view -o jsonpath='{.users[*].name}'    # get a list of users
```

Comment:
nit: how about 
```suggestion
kubectl config view -o jsonpath='{.users[*].name}'   # get a list of users
```
?

Reply:
Good idea. I noticed that there is whitespace issue in the other lines. I will change it in later commit.